src/main/java/com/rettichlp/unicacityaddon/events/faction/polizei/PlantBurnMessageEventHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.rettichlp.unicacityaddon.events.faction.polizei;
+
+import com.rettichlp.unicacityaddon.base.config.ConfigElements;
+import com.rettichlp.unicacityaddon.base.registry.annotation.UCEvent;
+import com.rettichlp.unicacityaddon.base.text.ColorCode;
+import com.rettichlp.unicacityaddon.base.text.Message;
+import com.rettichlp.unicacityaddon.base.text.PatternHandler;
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.regex.Matcher;
+
+/**
+ * @author Gelegenheitscode
+ */
+
+@UCEvent
+public class PlantBurnMessageEventHandler {
```

Comment:
Fass am besten beide EventHandler als FBIMessageEventHandler zusammen und erstell unter /faction ein neues Package /faction/fbi Package

Reply:
/burnplant ist ja für Polizei & FBI verfügbar, daher bewusst in Polizei gepackt